.github/workflows/ci.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - name: flutter test
         run: flutter test
```

Comment:
going forward (not for this PR), it may make sense to shard this testing script into two - one that tests the (2) dart leak tracker packages, and another that tests the flutter leak tracker packages.